components/portal/widgets/controllers.js

```diff
@@ -457,53 +459,56 @@ define(['angular'], function(angular) {
     };
 
     var resolveTemplateStatus = function(now, actionStart, actionEnd) {
-      var result = '';
+      if (!now || !actionStart || !actionEnd) {
```

Comment:
This is never true now, actionStart and actionEnd are invalid instead of null if they're missing from the config

Reply:
updated